build.gradle.kts

```diff
@@ -68,12 +68,6 @@ subprojects {
          * https://github.com/johnrengelman/shadow/blob/master/build.gradle
          */
 
-        // We need to do this otherwise the shadowJar tasks takes forever since
-        // it tries to shadow the entire gradle api
-        // Moreover, the gradleApi and groovy dependencies in the plugins themselves
-        // have been explicitly declared with the shadow configuration
-        configurations["compile"].dependencies.remove(dependencies.gradleApi())
```

Comment:
@geoand Usefully the latest version of the shadow plugin does this automatically :-)

Reply:
That's great! The less magic configuration the better!